dockers/docker-base/Dockerfile.j2

```diff
@@ -34,25 +34,45 @@ COPY ["etc/rsyslog.conf", "/etc/rsyslog.conf"]
 COPY ["etc/rsyslog.d/*", "/etc/rsyslog.d/"]
 COPY ["root/.vimrc", "/root/.vimrc"]
 
+# Install dependencies of supervisor
```

Comment:
in line 58, you have apt-get -y install -f that help to fix the dependency, so you probably do notneed line 38.

---
In reply to: [125733195](https://github.com/Azure/sonic-buildimage/pull/777#discussion_r125733195) [](ancestors = 125733195)